plugin/helpers.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package plugin
+
+import "github.com/mattermost/mattermost-server/model"
+
+type Helpers interface {
+	// EnsureBot ether returns an existing bot user or creates a bot user with
+	// the specifications of the passed bot.
+	// Returns the id of the bot created or existing.
+	EnsureBot(bot *model.Bot) (string, error)
+}
+
+type HelpersImpl struct {
```

Comment:
We don't need to export this, do we? I would rather keep it unexported.

Reply:
For the tests not to have an import cycle, yes.

Reply:
Could you elaborate a bit? `HelpersImpl` is only used in the package `plugin`. What am I missing?

Reply:
It's used in the plugin_test package in `plugin/helpers_bots_test.go` If we don't have that as a separate package, then when it tries to import the mocks for the API it fails because of the circular dependency.